arch/arm64/src/common/arm64_cpustart.c

```diff
@@ -231,6 +225,14 @@ void arm64_boot_secondary_c_routine(void)
   arm64_mmu_init(false);
 #endif
 
+  /* We need to confirm that current_task has been initialized. */
+
+  while (!current_task(this_cpu()));
+
+  /* Init idle task to percpu reg */
+
+  up_update_task(current_task(this_cpu()));
```

Comment:
do we need change other arch too

Reply:
Yes, the current community code implements percpu storage for this_task on arm64, armv7a, armv7r, and armv8r archs, all using this approach.